LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_GetPLCHostname.TcPOU

```diff
@@ -0,0 +1,45 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_GetPLCHostname" Id="{bd570dd7-dba7-4a18-a0ee-a97d90c7deaf}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_GetPLCHostname
+VAR_INPUT
+    fRetryDelay		: TIME := T#10s;
+    bExecute		: BOOL := TRUE;
+END_VAR
+VAR_OUTPUT
+    sHostname		: T_MaxString;
+    bDone 			: BOOL;
+    bError			: BOOL;
+END_VAR
+VAR
+    fbGetHostName	: FB_GetHostName := (bExecute := TRUE, sNetID := ''); //Acquires name of the PLC
```

Comment:
I don't think the initial value for bExecute here is ever used

Reply:
Addressed in poorly named commit 6fd976c